vignettes/traitstrap-workflow.Rmd

```diff
@@ -228,18 +290,48 @@ p_bootstrapped_moments <- trait_parametric_bootstrap(
 p_bootstrapped_moments
 ```
 
+The output of trait_parametric_bootstrap can be summarized using **trait_summarize_boot_moments** (see above).
+
+
+## Extracting raw distributions
+
+In traitstrap both the parametric and nonparametric boostrapping funcitons allow returing raw trait distributions.
+
+Note that if the data contains multiple traits, sites for several individuals, memory issues can occur and the raw trait distirbutions might have to be extracted from a subset of the data. 
```

Comment:
This should not be a problem now that the nrep is forced to one when raw == TRUE